addons/osio-addon/src/main/java/io/fabric8/launcher/osio/OsioLaunchMissionControl.java

```diff
@@ -74,6 +79,8 @@ public Boom launch(OsioLaunchProjectile projectile) {
 
         if (!projectile.isEmptyRepository()) {
```

Comment:
Can we get rid of this flag and call the analytics endpoint only if `projectDependencies().size() > 0`?

Reply:
No, Actually there can be a case where the user didn't add any dependencies. Then we have to go with core dependencies only.

Reply:
Then I think it's easier to invert this condition for clarity purposes.

Reply:
yeah, sure